cmd/manager/manager.go

```diff
@@ -61,9 +66,19 @@ var (
 	configPath           string
 	enableLeaderElection bool
 	featureGates         string
-	version              string
+	operatorVersion      string
 )
 
+func requireKubernetesVersion(version *version.Info, major, minor int) {
```

Comment:
We do not care about the patch versions, I assume?

Reply:
Yep.